client/next-web/src/app/_components/Tabs.js

```diff
@@ -0,0 +1,64 @@
+'use client'
```

Comment:
question, when should we use `use client`?

Reply:
When the component uses client-only React features and need to be imported into a Server component should add 'use client' . Once "use client" is defined in a file, all other modules imported into it, including child components, are considered part of the client bundle.

[nextJS doc](https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive)